### PR TITLE
Implement keylime test into openQA

### PR DESCRIPTION
**Pull request by rfan1**
Add the keylime attestation package.
Reference: https://github.com/keylime/keylime
Upstream status: Accepted (keylime >= 6.1.0)

- Related ticket: https://progress.opensuse.org/issues/106870
- Needles: n/a/
- Verification run: 

x86_64:
https://openqa.suse.de/tests/8178338# -SLE
https://openqa.opensuse.org/tests/2191560# -TW

aarch64:
https://openqa.suse.de/tests/8184248# -SLE
https://openqa.opensuse.org/tests/2193283# -TW

### CODE REVIEW

File: tests/security/tpm2/keylime/keylime.pm [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    zypper_call('in keylime-config keylime-firewalld keylime-agent keylime-tpm_cert_store keylime-registrar keylime-verifier', timeout => 240);
+
+    # Copy the keylime configuration file to /etc if not there
+    script_run("f=/etc/keylime.conf; [ ! -f \$f ] && cp /usr\$f \$f");
+
+    # Record the keylime packages' version for reference
+    my $pkgs_version = script_output('rpm -qa | grep keylime');
+    record_info("keylime packages version", "Current keylime packages' versoin: $pkgs_version");
+
+    # Make sure 'keylime_verifier' and 'keylime_registrar' services can be start successfully
```
Comment (ShawnHa0) [inline tests/security/tpm2/keylime/keylime.pm:29]: Typo here, should be 'can be started successfully'

File: tests/security/tpm2/keylime/keylime.pm [hunk 2]
```diff
@@ -0,0 +1,44 @@
+# Copyright 2020-2022 SUSE LLC
```
Comment (jouyingbin) [inline tests/security/tpm2/keylime/keylime.pm:1]: Is this a new case? maybe we can put 2022 only. how do you think?

**Comment by lilyeyes** [comment]
Discussed offline about the testing coverage, the PR generally LGTM.

**Review by lilyeyes — APPROVED** [review]
LGTM and please fix the CI error.

**Review by jouyingbin — COMMENTED** [review]
LGTM

**Pull request merged**